src/pages/workspaces/workspace/Data.js

```diff
@@ -129,216 +129,229 @@ const getReferenceData = _.flow(
   _.groupBy('datum')
 )
 
-const LocalVariablesContent = class LocalVariablesContent extends Component {
-  constructor(props) {
-    super(props)
-    this.state = {
-      editIndex: undefined,
-      deleteIndex: undefined,
-      editKey: undefined,
-      editValue: undefined,
-      editType: undefined,
-      textFilter: ''
-    }
+const LocalVariablesContent = ({ workspace, workspace: { workspace: { namespace, name } }, firstRender }) => {
```

Comment:
Clicking on 'Workspace Data' again while already on that sub-page no longer refreshes the workspace attributes. If we want to keep that behavior, we could pass `refreshKey` and trigger a reload when it changes. Also, we should stop passing `refreshWorkspace` since this is no longer used.